README.md

```
@@ -1 +1,104 @@
-# NordVPN-Easy-OpenWrt
\ No newline at end of file
+# NordVPN Easy for OpenWrt
+
+NordVPN Easy is a package-first NordVPN WireGuard integration for OpenWrt.
+The project is being built around native OpenWrt components: UCI
+configuration, init service integration, a LuCI frontend, and periodic
+health-check and recovery logic.
+
+### <ins>Project goals</ins>
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)